python/pyspark/ml/clustering.py

```diff
@@ -69,6 +70,25 @@ class KMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIter, HasTol
     True
     >>> rows[2].prediction == rows[3].prediction
     True
+    >>> import os, tempfile
```

Comment:
Sure thing will do

On Thursday, February 11, 2016, Xiangrui Meng <notifications@github.com>
wrote:

> In python/pyspark/ml/clustering.py
> <https://github.com/apache/spark/pull/10999#discussion_r52688040>:
>
> > @@ -69,6 +70,25 @@ class KMeans(JavaEstimator, HasFeaturesCol, HasPredictionCol, HasMaxIter, HasTol
> >      True
> >      >>> rows[2].prediction == rows[3].prediction
> >      True
> > +    >>> import os, tempfile
>
> I agree with @holdenk <https://github.com/holdenk> . This may be too
> verbose for a doctest. We can move the temp directory setup in test
> preparation (where we initialize sqlContext) and clean up. We can do that
> in a separate PR. @holdenk <https://github.com/holdenk> Could you create
> a JIRA for it? Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/10999/files#r52688040>.
>


-- 
Cell : 425-233-8271
Twitter: https://twitter.com/holdenkarau